PhysicsTools/NanoAOD/python/muons_cff.py

```diff
@@ -153,6 +181,11 @@
     modifier.toModify(muonTable.variables, puppiIsoId = None, softMva = None)
 
 run2_nanoAOD_102Xv1.toModify(muonTable.variables, puppiIsoId = None)
+print(modifier)
+for modifier in (run2_miniAOD_80XLegacy,run2_nanoAOD_92X,run2_nanoAOD_94X2016,run2_nanoAOD_94XMiniAODv1,run2_nanoAOD_94XMiniAODv2,run2_nanoAOD_102Xv1,run2_nanoAOD_106Xv1,run2_nanoAOD_106Xv2,run3_nanoAOD_122):
+    modifier.toModify(muonTable.variables,mvaIDMuon=None )
+    modifier.toModify(muonTable.variables,mvaIDMuon_WP=None )
+    modifier.toModify(muonTable.externalVariables, mvaIDMuon = ExtVar(cms.InputTag("muonMVAID:probGOOD"),float, doc="MVA-based ID score (from nanoAOD)",precision=14))
```

Comment:
- Similar comments apply here: you can remove "(from nanoAOD)" from the docstring, and probably reduce the precision
- In this case you only store the MVA value and not the working point, is that expected?